xmtp_id/src/associations/mod.rs

```diff
@@ -569,10 +557,7 @@ mod tests {
 
         let revoke_result =
             apply_update(new_state, IdentityUpdate::new_test(vec![attempted_revoke]));
-        assert!(revoke_result.is_err());
-        assert_eq!(
-            revoke_result.err().unwrap(),
-            AssociationError::MissingExistingMember
-        );
+        assert!(revoke_result
```

Comment:
we can make use of the `matches!` macro here instead of turning into a string and comparing.

`assert!(matches!(result, Err(AssociationError::MissingExistingMember))`